docs/developers/celestia-node.md

```diff
@@ -10,6 +10,36 @@ environment [here](./environment.md).
 
```

Comment:
Should we have any information here between the differences of Arabica vs Mamaki?

Reply:
Or maybe just link to the existing info?

Reply:
check out the new `participate.md` page